src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -2126,6 +2126,9 @@ export default {
 		duplicateUsername: "Username '%0%' is already taken",
 		legacyOption: 'Legacy option',
 		legacyOptionDescription: 'This option is no longer supported, please select something else',
+		numberMinimum: "Value must be greater than or equal to '%0%'.",
+		numberMaximum: "Value must be less than or equal to '%0%'.",
+		numberMisconfigured: "Minimum value '%1%' must be less than the maximum value '%0%'.",
```

Comment:
Looks like these tokens are the wrong way around - you are passing in minimum first so should be `%0%`.

Reply:
Doh! 🤦 I'll get that fixed up shortly.